src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -64,8 +71,14 @@ public override void Dispose()
     public class FamilyTypes : RevitDropDownBase
     {
         private const string NO_FAMILY_TYPES = "No family types available.";
+        private const string outputName = "Family Type";
+
+        public FamilyTypes() : base(outputName) { }
 
-        public FamilyTypes() : base("Family Type") { }
+        [JsonConstructor]
```

Comment:
broke out these hardcoded `outputName` hardcoded strings to class constants

Reply:
this applies to all the dropdown nodes (selection is a bit different - see below)